MONAIAuto3DSeg/Resources/Models.json

```
@@ -81,6 +81,16 @@
                 { "url": "https://github.com/lassoan/SlicerMONAIAuto3DSeg/releases/download/Models/mediastinal-anatomy-3mm-v2.0.0.zip" }
             ]
         },
+        {
+            "title": "Prostate - Multisequence",
+            "description": "Segments peripheral (PZ) and transition zone (TZ) on T2 and ADC sequences. Trained on Medical Segmentation Decathlon dataset",
+            "subject": "human",
+            "imagingModality": "MRI",
```

DICOM standard for imaging modality is `MR`:
```suggestion
            "imagingModality": "MR",
```